shared/chat/conversation/messages/message-popup/header.js

```diff
@@ -41,14 +41,18 @@ const MessagePopupHeader = (props: {
       style={{
         ...globalStyles.flexBoxColumn,
         alignItems: 'center',
+        maxWidth: isMobile ? '100%' : 240,
+        paddingLeft: globalMargins.small,
+        paddingRight: globalMargins.small,
+        textAlign: 'center',
```

Comment:
I bet @chrisnojima would prefer the use of Box2 rather than adding to an old box style